types/rational.go

```diff
@@ -144,7 +144,7 @@ func (r Rat) Quo(r2 Rat) Rat      { return Rat{new(big.Rat).Quo(r.Rat, r2.Rat)}
 func (r Rat) Add(r2 Rat) Rat      { return Rat{new(big.Rat).Add(r.Rat, r2.Rat)} } // Add - addition
 func (r Rat) Sub(r2 Rat) Rat      { return Rat{new(big.Rat).Sub(r.Rat, r2.Rat)} } // Sub - subtraction
 func (r Rat) String() string      { return r.Rat.String() }
-func (r Rat) FloatString() string { return r.Rat.FloatString(18) } // a human-friendly string format. The last digit is rounded to nearest, with halves rounded away from zero.
+func (r Rat) FloatString() string { return r.Rat.FloatString(28) } // a human-friendly string format. The last digit is rounded to nearest, with halves rounded away from zero.
```

Comment:
Now delegation shares will have 10 decimals, and to make it readable, we just divide it with 10^18, so it may have 28 decimals.